exercises/transpose/example.jl

```diff
@@ -0,0 +1,20 @@
+function transpose_strings(input::AbstractArray)
+    cols = length(input)
+    cols == 0 && return []
+    rows = maximum(map(x->length(x), input))
+    result = fill("", rows)
+    for i = 1:rows
+        for j = 1:cols
+            sym = ""
+            if i <= length(input[j])
+                delta = j-length(result[i])-1
+                if delta > 0
+                    sym = " " ^ delta
+                end
+                sym = sym * string(input[j][i])
```

Comment:
`sym *= string(...)` would work as well, looks a bit cleaner imo but that's personal taste.